src/node_sqlite.h

```diff
@@ -21,7 +21,8 @@ class DatabaseSync : public BaseObject {
   DatabaseSync(Environment* env,
                v8::Local<v8::Object> object,
                v8::Local<v8::String> location,
-               bool open);
+               bool open,
+               bool enable_foreign_keys_on_open);
```

Comment:
non-blocking nit: totally not a fan of bool arguments like this. Would much prefer having either an options struct or enums for this purpose.

Reply:
@jasnell I agree, I am planning to add at least one more option and to then wrap the `*_on_open` options in a `struct`.